COMPILING.md

```diff
@@ -0,0 +1,140 @@
+
+Note that you don’t need to install the JACK package(s) for a headless build. If you plan to run headless on Gentoo, or are compiling under Ubuntu for use on another Ubuntu machine, the only packages you should need for a headless build are `qtcore`, `qtnetwork`, `qtconcurrent` and `qtxml` (both for building and running the server).
+
+Compile the sources to ignore the JACK sound library:
+
+```
+make distclean # recommended
+qmake "CONFIG+=nosound headless"
+make
+sudo make install # optional
+```
+
+To control the server with systemd, see this [unit file example](https://github.com/jamulussoftware/jamulus/blob/master/distributions/jamulus-server.service). See also runtime [configuration options](/wiki/Command-Line-Options), and [this information](/wiki/Tips-Tricks-More#controlling-recording-on-linux-headless-servers) on controlling recordings on headless servers.
+
+---
+
+## Windows
+
+
+You will need [Qt](https://www.qt.io/download)
+
+* Use the free GPLv2 license for Open Source development
+* To determine the Qt version you need, check [qt-installer-windows.qs](https://github.com/jamulussoftware/jamulus/blob/master/windows/qt-installer-windows.qs): under INSTALL_COMPONENTS you will see `qt.qt5.[version]`, e.g., 5123 means version 5.12.3.
+* Select Components during installation: Expand the Qt section, find the matching version, e.g., **Qt 5.12.3**, and add the compiler components for your compiler, e.g., `MSVC 2017 32-bit/64-bit` for Visual Studio 2019
+* [ASIO development files](https://www.steinberg.net/en/company/developer.html)
+
+### Compiling and building installer
+
+Most users will probably want to use this method:
+
+1. Open PowerShell
+1. Navigate to the `jamulus` directory
+1. To allow unsigned scripts, right-click on the `windows\deploy_windows.ps1` script, select properties and allow the execution of this script. You can also run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. (You can revert this after having run this script. For more information see the [Microsoft PowerShell documentation page](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy)).
+1. Edit the $QtCompile32 and $QtCompile64 variables.
+1. Run the Jamulus compilation and installer script in PowerShell: `.\windows\deploy_windows.ps1`.
+1. You can now find the Jamulus installer in the `.\deploy` directory.
+
+### Compiling only
+
+1. Create a folder under `\windows` called ASIOSDK2
+1. Download the [ASIOSDK](https://www.steinberg.net/asiosdk), open the top level folder in the .zip file and copy the contents into `[\path\to\jamulus\source]\windows\ASIOSDK2` so that, e.g., the folder `[\path\to\jamulus\source]\windows\ASIOSDK2\common` exists
+1. Open Jamulus.pro in Qt Creator, configure the project with a default kit, then compile & run
+
+---
+
+## macOS
+You will need XCode and Qt as follows:
+
```

Comment:
Sorry to be a pain on this... :)  If we're moving off `brew` for the automation, should we move off it here, too?  @softins 

(I was also thinking if we should recommend `aqtinstall` generally for installing Qt, so these instructions mirror the autobuild more closely.)